### DISCUSSION TITLE
Managing user resource with tenacy feature

### DISCUSSION THREAD

**Discussion by hannanmiah** [filamentphp/filament#7870] [Help]
How can I manage user resource through filament tenacy feature? If there has many to many relationship between Team and User model , I want to manage every team users separately. Due to many to many relation my users associate with other teams also has access. I want restrict this access and I should be the only managable person of my users. How can I achieve this? Any Idea? 
#help @danharrin

**Comment by Z3d0X** [comment]
I haven't used tenancy features. However to answer:

> I want restrict this access and I should be the only managable person of my users. 

https://filamentphp.com/docs/3.x/panels/resources/getting-started#authorization

Another idea is to create a new panel without tenancy & set it up so that only you can access it
https://filamentphp.com/docs/3.x/panels/users#authorizing-access-to-the-panel

**Reply by hannanmiah** [reply]
I want to make this feature for every tenant so that he/they can own their users individually.

**Comment by micraux** [comment]
You could use a role/permission package... or maybe just a new column in your users table (like a boolean IS_TEAM_MANAGER) and then set the authorization feature as  @Z3d0X said
